### ISSUE TITLE
Allow decimal.localcontext to accept keyword arguments to set context attributes

### ISSUE THREAD

**Issue by stevendaprano** [python/cpython#91291]
BPO | [47135](https://bugs.python.org/issue47135)
--- | :---
Nosy | @ncoghlan, @stevendaprano, @corona10, @dignissimus
PRs | <li>python/cpython#32242</li>
Files | <li>[sam_ezeh.patch](https://bugs.python.org/file50713/sam_ezeh.patch "Uploaded as text/plain at 2022-04-01.19:30:57 by @dignissimus")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2022-03-27.10:16:04.212>
labels = ['library', '3.11']
title = 'Allow decimal.localcontext to accept keyword arguments to set context attributes'
updated_at = <Date 2022-04-01.20:49:19.136>
user = 'https://github.com/stevendaprano'
```

bugs.python.org fields:
```python
activity = <Date 2022-04-01.20:49:19.136>
actor = 'sam_ezeh'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2022-03-27.10:16:04.212>
creator = 'steven.daprano'
dependencies = []
files = ['50713']
hgrepos = []
issue_num = 47135
keywords = ['patch']
message_count = 6.0
messages = ['416114', '416364', '416507', '416511', '416513', '416514']
nosy_count = 4.0
nosy_names = ['ncoghlan', 'steven.daprano', 'corona10', 'sam_ezeh']
pr_nums = ['32242']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue47135'
versions = ['Python 3.11']
```

</p></details>

**Comment by JelleZijlstra** [comment]
Implemented in #32242. Thanks for the PR!

**Comment by jkloth** [comment]
It appears this change is also causing issues on the entire buildbot fleet
